src/vs/workbench/browser/parts/editor/noTabsTitleControl.ts

```diff
@@ -50,6 +50,8 @@ export class NoTabsTitleControl extends TitleControl {
 		this.editorLabel = this._register(this.instantiationService.createInstance(ResourceLabel, labelContainer, undefined)).element;
 		this._register(addDisposableListener(this.editorLabel.element, EventType.CLICK, e => this.onTitleLabelClick(e)));
 
+		this.updateTabHeight();
```

Comment:
Since this is also called from the tabs title control, we can probably move this into the parent `titleControl.ts` method and then make sure to call `super.create`